aspnetcore/tutorials/build-your-first-blazor-app.md

```diff
@@ -19,6 +19,15 @@ This tutorial shows you how to build and modify a Blazor app.
 
 Follow the guidance in the <xref:blazor/get-started> article to create a Blazor project for this tutorial. Name the project *ToDoList*.
 
+## Blazor app files
+
+Blazor apps use the following special files and components:
+
+* [\_Host.cshtml](xref:blazor/routing#aspnet-core-endpoint-routing-integration) &ndash; A Razor page where the app's components are rendered.
+* [\_Imports.razor](xref:blazor/layouts#centralized-layout-selection) &ndash; Imports namespaces for Razor components.
+* [App.razor](xref:blazor/routing#route-templates) &ndash; Maintains the `Router` component, which enables routing to the app's components.
```

Comment:
Let's also include something like "This is the root components, which will host the rest of the components of your application".